src/platforms/java/guides/spring-boot/webflux.mdx

```diff
@@ -0,0 +1,11 @@
+---
+title: Webflux Integration
+sidebar_order: 6
+description: "Learn more about using Spring Webflux with Sentry."
+---
+
+Similar to Spring MVC, Sentry has a support for Spring Webflux. When developing a reactive web application, Sentry Spring Boot Starter creates infrastructure required for capturing errors triggered during HTTP request processing.
+
+<Note>
+Webflux support is available from version 5.1.0 in experimental phase. The API as well as internals may change in the future releases.
```

Comment:
that's fine, but we have a "predefined" wording for that (its part of our docs guidelines):

> Support for Flutter's Desktop is available as a Beta. Features available in Beta are still in-progress and may have bugs. We recognize the irony.

we can either adapt the message above or keep it as it is, your call.

Reply:
I'll adapt